netplay/joinroom.cpp

```diff
@@ -102,14 +103,15 @@ void JoinRoom::resetList()
     row = 0;
     rooms.clear();
     listWidget->clear();
-    listWidget->setColumnCount(5);
+    listWidget->setColumnCount(6);
     listWidget->setRowCount(row);
     QStringList headers;
     headers.append("Room Name");
     headers.append("Game Name");
     headers.append("Game MD5");
     headers.append("Password Protected");
     headers.append("LLE");
+    headers.append("Input Delay");
```

Comment:
Can we rename this to "Fixed Input Delay"? It gets a "yes" or "no" value, so I want to make it clear that it's talking about having fixed input delay enabled